migrations/versions/0025_notify_service_data.py

```diff
@@ -0,0 +1,119 @@
+"""empty message
+
+Revision ID: 0025_notify_service_data
+Revises: 0024_add_research_mode_defaults
+Create Date: 2016-06-01 14:17:01.963181
+
+"""
+
+# revision identifiers, used by Alembic.
+from datetime import datetime
+
+from alembic import op
+
+from app.encryption import hashpw
+import uuid
+revision = '0025_notify_service_data'
+down_revision = '0024_add_research_mode_defaults'
+
+
+user_id= '6af522d0-2915-4e52-83a3-3690455a5fe6'
+service_id = 'd6aa2c68-a2d9-4437-ab19-3ae8eb202553'
+
+
+def upgrade():
+    password = hashpw(str(uuid.uuid4()))
+    op.get_bind()
+    user_insert = """INSERT INTO users (id, name, email_address, created_at, failed_login_count, _password, mobile_number, state, platform_admin)
+                     VALUES ('{}', 'Notify service user', 'this-does-not-work@this.does.not.work.gov.uk', '{}', 0,'{}', '+441234123412', 'active', False)
```

Comment:
Can we make the email address more instructive of the reason it exists (notify-service-user@digital.cabinet-office or something. Might not work, but when I'm scanning the DB in a year might be useful.